### PR TITLE
Docs: small fixes

### PR DESCRIPTION
**Pull request by shykes**
* Clarify a title
* Move 0.1 migration to guides
* Remove duplicate sidebars entry

### CODE REVIEW

**Review by jlongtine — APPROVED** [review]

File: docs/guides/1216-docker-cli-load.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ slug: /1216/docker-cli-load
 displayed_sidebar: '0.2'
 ---
 
-# Loading a dagger image into a docker daemon
+# Loading an image into a docker engine
 
 Using `cli.#Load`, you can save a dagger image (`docker.#Image`) into a local or remote engine.
```
Comment (sipsma) [inline docs/guides/1216-docker-cli-load.md:8]: nit: Not sure if the intent was to change all occurrences of `dagger image` to just `image` or just the title, but there is another occurrence here.

**Review by sipsma — APPROVED** [review]
One nit/question, otherwise LGTM

File: docs/guides/1216-docker-cli-load.md (hunk 1 above)
Comment (shykes) [reply docs/guides/1216-docker-cli-load.md:8]: Oops, not intended, I just didn't think to check the text :) Will fix, thanks.

Comment (shykes) [reply docs/guides/1216-docker-cli-load.md]: Fixed, I ended up making more edits to the text while I was at it.

**Review by sipsma — APPROVED** [review]
LGTM

File: website/sidebars.js [hunk 2]
```diff
@@ -53,8 +53,8 @@ module.exports = {
         "guides/custom-buildkit",
         "guides/self-signed-certificates",
         "guides/pushing-plan-dependencies",
-        "guides/coding-style",
```
Comment (gerhard) [inline website/sidebars.js:56]: I just spotted this one live, this was a nice surprise 😉

**Review by gerhard — APPROVED** [review]
🚀

**Pull request merged**